app/tests/api/team-api.test.js

```diff
@@ -241,3 +242,38 @@ test('remove moderator from team', async t => {
   const matchOsmIdAssigned = data => data.osm_id === osmId
   t.false(any(matchOsmIdAssigned, moderators))
 })
+
+test('get my teams list', async t => {
+  // create two teams (osmId = 1)
+  await agent.post('/api/teams')
+    .send({ name: 'map team ♾+2' })
+    .expect(200)
+  const team = await agent.post('/api/teams')
+    .send({ name: 'map team ♾+3' })
+    .expect(200)
+  const { id: teamId } = team.body
+  // add osmId 2 to one team, and make them moderator
+  await agent.put(`/api/teams/add/${teamId}/2`)
+    .expect(200)
+  await agent.put(`/api/teams/${teamId}/assignModerator/2`)
+    .expect(200)
+  // remove osmId 1 from moderator
+  await agent.put(`/api/teams/${teamId}/removeModerator/1`)
+    .expect(200)
+  // check myTeams listing
+  const response = await agent.get(`/api/my/teams`)
+    .expect(200)
+  const { osmId, member, moderator } = response.body
```

Comment:
@guidorice should we make these exact numbers? as in, moderator of 1 team, and member of 2 teams?

Reply:
@kamicut yes that is what I had done originally, but I noticed that all the other teams from `team-api.test.js` are still in the database. I kind of punted instead of spending the time to start with a clean collection of teams. What do you think?

Reply:
We discussed on the scrum call and I agree these team counts should be exact #s. Will fix 👍